website/src/components/CommunitySlideCarouselSection/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Carousel from 'better-react-carousel';
+import features from '../../../websiteData/features';
+import CommunitySlide from '../CommunitySlide';
+
+const CommunitySlideCarouselSection = () => {
+  return (
+    <>
+      <Carousel
+        responsiveLayout={[
```

Comment:
Create a constant array containing details about responsive layout for Carousel and pass that variable as a prop here.